corehq/apps/accounting/tests/test_stripe_payment.py

```diff
@@ -1,22 +1,28 @@
 from __future__ import absolute_import
 from mock import patch
 
-from django.test import TransactionTestCase
+from django.test import TestCase
 from django.test.client import RequestFactory
 
 import stripe
+from stripe.resource import StripeObject
 
 from corehq.apps.accounting.models import PaymentRecord, PaymentMethod, BillingAccount
 from corehq.apps.accounting.payment_handlers import CreditStripePaymentHandler
-from corehq.apps.accounting.tests import generator
 from corehq.apps.domain.models import Domain
 
 
-class TestCreditStripePaymentHandler(TransactionTestCase):
+class MockFailingStripeObject(object):
+
+    @property
+    def id(self):
+        raise Exception
+
+
+class TestCreditStripePaymentHandler(TestCase):
```

Comment:
awesome, thanks!

> I don't think this is needed since the default self.tearDown() should rollback any SQL objects created.

Fancy!  So that's anything created during the test, but leaving stuff created in `setUpClass` alone?